utilities/NotificationHelper.js

```diff
@@ -79,11 +79,11 @@ class NotificationHelper {
         const start_time_list = NC_START.split(':');
         const end_time_list = NC_END.split(':');
         const now = new Date();
-        const hr = now.getHours();
-        const min = now.getMinutes();
-        return ((hr < end_time_list[0]) || ((hr == end_time_list[0]) &&
-            (min < end_time_list[0]))) && ((hr > start_time_list[0]) ||
-            ((hr == start_time_list[0]) && (min > start_time_list[0])));
+        const start = new Date();
+        start.setHours(parseInt(start_time_list[0]), parseInt(start_time_list[1]), 0);
+        const end = new Date();
+        end.setHours(parseInt(end_time_list[0]), parseInt(end_time_list[1]), 0);
+        return !(now < start && now > end);
```

Comment:
What is being checked here is if now is now is before the start of tonight's crew and after the end of last nights crew and then "not"ing it.  Basically are we not not in night crew.

Logically this is equivalent to (now > start || now < end).  What is being checked here is if now is before the end of last night's crew or after the start of tonight's crew.

Reply:
Yes thanks. XD I swear I had that combination previously, and it didn't work (since I was using different way of calculating time). Must have forgot to change back. 👍